app/controllers/users_controller.rb

```diff
@@ -40,6 +44,7 @@ def create
   # PATCH/PUT /users/1
   # PATCH/PUT /users/1.json
   def update
```

Comment:
We could remove create and destroy from this file. We does not really need those actions. Also we could remove the routes to them.